7_deploy_osh/roles/suse-dev-patcher/tasks/main.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Install system packages for dev patcher
+  package:
+    name: "{{ suse_dev_patcher_packages }}"
+    state: present
+  register: install_packages
+  until: install_packages is success
+  retries: 5
+  delay: 2
+  tags:
+    - install
+
+- name: Clone OSH related repos
+  include: clone-repos.yml
```

Comment:
`include` should probably replaced with a `import_tasks` here.

Reply:
Okay..will change it here.